x-pack/plugins/observability/public/hooks/slo/use_fetch_slo_list.ts

```diff
@@ -19,6 +19,7 @@ import {
 
 import { useKibana } from '../../utils/kibana_react';
 import { sloKeys } from './query_key_factory';
+import { mixKqlWithTags } from './mix_kql_with_tags';
```

Comment:
I extracted this to a helper function, since I am gonna need it in the slo grouping as well. Not sure if this is the right place though.